Makefile

```diff
@@ -42,7 +42,8 @@ build-cli: output-dir ## Build the nginx-meshctl binary
 
 .PHONY: test
 test: output-dir ## Run unit tests for the Go code
-	go test ./... -race -coverprofile=$(OUTPUT_DIR)/coverage.out
+	go test ./... -race -coverprofile=$(OUTPUT_DIR)/coverage.out \
+		-test.parallel 4
```

Comment:
Couple things:
1. I believe this is a flag meant to be run by the go test binary (which is compiled and set by the go test command itself when run), so we would actually want to use `-parallel`.
2. By default, this value is set per machine based on the GOMAXPROCS env var, so hard setting this could reduce performance for certain people with a higher core count.

Is there a specific reason you set this value?